### ISSUE TITLE
Access tun device inside docker container

### ISSUE THREAD

**Issue by bbirand** [jpetazzo/pipework#35]
Hi,

Thanks a lot for pipework, it is truly very useful! I have an unusual setup, in which I am running an OpenVPN client inside a docker container. This creates a tun device inside the container. I now want to bridge this tun interface with a bridge that exists on the host device, in order to transmit data out of this VPN connection. Is this possible? 

Thanks!

**Comment by jpetazzo** [comment]
Closing older issues.

**Comment by hugochinchilla** [comment]
@jpetazzo I'm configuring a setup like the one you describe, I can ping other clients in the VPN from both the container and the host, but in addition I would like to be able to ping from other machines using the host as the gateway, I've setup a route from other machine but it's not enough, i think i need to add some iptables rule in the host but my iptables wizardry is very limited and i didn't manage to make it work, any sugestions?

**Comment by jpetazzo** [comment]
Hi @hugochinchilla,

Could you open a new issue with that question, please?

Thank you!

**Comment by hugochinchilla** [comment]
It's not really an issue with pipework, i'm not using it, just asking for advice, I found this when googling and looks a lot like my setup.

**Comment by jpetazzo** [comment]
Ah, if you're not using pipework, I'm sorry, this is probably not the right venue for your question ... Maybe you could try on stackoverflow? Sorry again!

**Comment by valentt** [comment]
@jpetazzo I tried your recommendation and I still can't ping openvpn IP adresses from host.
My docker0 ip from host side is 172.17.0.1, inside docker etho has 172.17.0.2 address. OpenVPN uses 10.1.1.0/24 network so inside docker I enabled masquerade on tun0 like this:
iptables -t nat -A POSTROUTING -o tun0 -j MASQUERADE

and on host added route:
ip route add 10.1.1.0/24 via 172.17.0.1

and I also tried:
ip route add 10.1.1.0/24 via 172.17.0.2

I still can only ping 10.1.1.1 from inside docker but not from host :(

Any ideas why?
